packages/app/src/constants/paths.ts

```diff
@@ -17,7 +17,7 @@ export enum Paths {
   RECOVER = "/recover",
   RESET_PASSWORD = "/reset-password",
   ADD_VERIFIABLE_CREDENTIAL = "/add-verifiable-credential",
-  GENERATE_VERIFIABLE_PRESENTATION_REQUEST = "/generate-verifiable-presentation-request",
+  GENERATE_VERIFIABLE_PRESENTATION = "/generate-verifiable-presentation",
```

Comment:
```suggestion
  GENERATE_VP = "/generate-vp",
```

Reply:
Same as above, I think this should be the full name.